### ISSUE TITLE
`with_crate` panics on `0.8.5`

### ISSUE THREAD

**Issue by elichai** [eqrion/cbindgen#335]
Hi,
After the new update all my crates panic, I narrowed it down to this example:
```
use cbindgen;
use std::env;
fn main() {
    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
    let a = cbindgen::Builder::new()
        .with_crate(&crate_dir)
        .generate();
}
```

with `cbindgen = "=0.8.4"` it compiles just fine.
and with `cbindgen "=0.8.5"` it panics:
```
--- stderr
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:345:21
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:70
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:58
             at src/libstd/panicking.rs:200
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:215
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:478
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:385
   6: rust_begin_unwind
             at src/libstd/panicking.rs:312
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::panicking::panic
             at src/libcore/panicking.rs:49
   9: <core::option::Option<T>>::unwrap
             at /rustc/fc50f328b0353b285421b8ff5d4100966387a997/src/libcore/macros.rs:11
  10: cbindgen::bindgen::cargo::cargo::Cargo::dependencies
             at /home/elichai2/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.8.5/src/bindgen/cargo/cargo.rs:88
  11: cbindgen::bindgen::parser::Parser::parse_crate
             at /home/elichai2/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.8.5/src/bindgen/parser.rs:162
  12: cbindgen::bindgen::parser::parse_lib
             at /home/elichai2/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.8.5/src/bindgen/parser.rs:102
  13: cbindgen::bindgen::builder::Builder::generate
             at /home/elichai2/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.8.5/src/bindgen/builder.rs:322
  14: build_script_build::main
             at ./build.rs:7
  15: std::rt::lang_start::{{closure}}
             at /rustc/fc50f328b0353b285421b8ff5d4100966387a997/src/libstd/rt.rs:64
  16: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:297
  17: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  18: std::rt::lang_start_internal
             at src/libstd/panicking.rs:276
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  19: std::rt::lang_start
             at /rustc/fc50f328b0353b285421b8ff5d4100966387a997/src/libstd/rt.rs:64
  20: main
  21: __libc_start_main
  22: _start
```

**Comment by mbrobbel** [comment]
Looks like [this](https://github.com/eqrion/cbindgen/commit/7efea2912056f67271468684c52b2e9d7d7b94b6#diff-74a07aadbae621b980e27a84a86295f3L112) check no longer exists in the [dependencies](https://github.com/eqrion/cbindgen/commit/7efea2912056f67271468684c52b2e9d7d7b94b6#diff-74a07aadbae621b980e27a84a86295f3L91) method, called [here](https://github.com/eqrion/cbindgen/commit/7efea2912056f67271468684c52b2e9d7d7b94b6#diff-50c7afb4d0496221f37158c6a0972400R162).

**Comment by emilio** [comment]
cc @Gankro

**Comment by emilio** [comment]
Fixed in https://github.com/eqrion/cbindgen/commit/6d9d395cd4d57468158259be24d78f4d7efcff4d. I verified that enigma-types generates the same header as with 0.8.4. We don't have a pre-existing integration-test for this case, but I'll try to add one if I find some time.

**Comment by Gankro** [comment]
oops, sorry 😬

**Comment by elichai** [comment]
@emilio Thank you! i'll remove the version locking after the next version release.
(maybe 0.8.5 should be yanked?)

**Comment by eqrion** [comment]
@elichai  I've published 0.8.6 with the fix. 0.8.5 has also been yanked.
